src/game.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const readline = require('readline');
+const getRandomNumber = require('./randomizer');
+const checkCattle = require('./checkCattle');
+const myConsole = require('console');
+
+const win = `
+              (      )
+              ~(^^^^)~
+               ) @@ \\~_          |\\
+              /     | \\        \\~ /
+             ( 0  0  ) \\        | |
+              ---___/~  \\       | |
+               /'__/ |   ~-_____/ |
+o          _   ~----~      ___---~
+  O       //     |         |
+         ((~\\  _|         -|
+   o  O //-_ \\/ |        ~  |
+        ^   \\_ /         ~  |
+               |          ~ |
+               |     /     ~ |
+               |     (       |
+                \\     \\      /\\
+               / -_____-\\   \\ ~~-*
+               |  /       \\  \\
+               / /         / /
+             /~  |       /~  |
+             ~~~~        ~~~~
+`;
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+async function game() {
+  myConsole.log('፨ Buls and cows ፨');
+
+  const number = getRandomNumber(1000, 9999).toString();
+  let isEnd = false;
+
+  while (!isEnd) {
+    try {
+      await turn(number);
+    } catch (err) {
+      isEnd = true;
+    }
+  }
+}
+
+async function turn(number) {
+  return new Promise((resolve, reject) => {
+    terminal.question('Gues my 4 digit number: ', (userInput) => {
+      if (userInput === number) {
+        myConsole.log('You won!!!', win);
+        terminal.close();
+        reject(new Error());
```

Comment:
That's good that you used Promise here, but why rejecting when the user wins and then catching this to stop the game? That's confusing... you can resolve with boolean value true/false and then in
`isEnd = await turn(number);`